src/store/accountStore.js

```diff
@@ -9,7 +11,7 @@ export default {
     doubleName: false
   },
   actions: {
-    login (context, userData) {
+    async login (context, userData) {
```

Comment:
Has impact on src/store/authStore.js:69 and src/store/authStore.js:139
needs change?

Reply:
authStore.js needs refactor anyways, is not used atm.